content/hardware/04.pro/boards/portenta-x8/tutorials/datalogging-iot/content.md

```diff
@@ -0,0 +1,416 @@
+---
+title: 'Data Logging with MQTT, Node-RED, InfluxDB and Grafana'
+description: 'This tutorial will show you how to set up a local data logging application using an MQTT broker, Node-RED, InfluxDB, Grafana, and the Arduino® Portenta X8.'
+difficulty: Intermediate
+tags:
+  - Docker
+  - MQTT
+  - Mosquitto
+  - Node-RED
+  - InfluxDB
+  - Grafana
+author: 'José Bagur, Taddy Chung'
+software:
+  - Terminal
+  - Docker
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+This tutorial will set up a typical Internet of Things (IoT) application: an MQTT data logger. For this application, we are going to use what we call in the Arduino team the "IoT-Quartet," four common building blocks that are popular and regularly used in these types of applications:
+
+- Mosquitto (MQTT broker) 
+- Node-RED
+- InfluxDB
+- Grafana

```

Comment:
I would add a link to their documentation site or the project site

Reply:
Done!